apps/site/next.dynamic.mjs

```diff
@@ -208,6 +208,10 @@ const getDynamicRouter = async () => {
       ? `${siteConfig.title} — ${data.title}`
       : siteConfig.title;
 
+    if (data.description) {
```

Comment:
No need of an if statement, if it is undefined it will just... return nothing. Also pelase update the types so that description is a field it recognizes as part of Frontmatter (on our types folder)